src/components/lastRestaurants/lastRestaurants.css

```diff
@@ -0,0 +1,61 @@
+.last-restaurants {
+  padding-top: 20px;
+  width: 100%;
+}
+.last-restaurants__title-name {
+  color: #3f3e3e;
+  padding: 20px 0 20px 86px;
+}
+.last-restaurants__container {
+  padding: 20px 0 20px 86px;
```

Comment:
its really necessary this padding so big?
you can use     margin: 0 auto; and the content should be in the center of the page


I changed here:
<img width="1420" alt="image" src="https://user-images.githubusercontent.com/38358239/169145524-60cba910-91b6-4a57-89c1-d675688ac156.png">